CRM/Utils/Check/Component/Security.php

```diff
@@ -71,7 +71,7 @@ public function checkLogFileIsNotAccessible() {
           $url[] = $log_path[1];
           $log_url = implode($filePathMarker, $url);
           if ($this->fileExists($log_url)) {
-            $docs_url = $this->createDocUrl('checkLogFileIsNotAccessible');
```

Comment:
If I enter https://wiki.civicrm.org/confluence/display/CRMDOC//uploads-should-not-be-accessible I get a 404 - unchanged I get redirected to the security page on the docs - ie the equivalent of

return CRM_Utils_System::docURL2('sysadmin/setup/security#' . $topic, TRUE);

So this change make it un-better

Reply:
isn't that what L367 is meant to be doing?)